### PR TITLE
CLDR-14610 add Tok Pisin (tpi) to seed

### PR DESCRIPTION
**Pull request by DavidLRowe**
CLDR-14610

- [ X ] This PR completes the ticket.

### CODE REVIEW

**Review by srl295 — APPROVED** [review]

**Comment by srl295** [comment]
@DavidLRowe congrats on your merge 🎉

File: common/supplemental/languageGroup.xml [hunk 1]
```diff
@@ -34,6 +34,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<languageGroup parent="cdd">ari cad kii paw wic</languageGroup>
 		<languageGroup parent="cel">br cy ga gd gv kw mga nrc obt owl sga wlm xbm xcb xce xga xpi</languageGroup>
 		<languageGroup parent="cmc">ace cje cjm hro huq jra rad rog</languageGroup>
+		<languageGroup parent="cpe">tpi</languageGroup>
```
Comment (srl295) [inline common/supplemental/languageGroup.xml:37]: just remembered that this file is _generated_ but it can be checked when that BRS step is reached.

**Comment by DavidLRowe** [comment]
What is the source from which this file is generated?

**Comment by srl295** [comment]
> What is the source from which this file is generated?

huh… looks like the docs didn't get updated properly …tools updated in #900 but blocked by #666

I'm trying this command: mvn -DCLDR_DIR=${HOME}/src/cldr3 -Dexec.mainClass=org.unicode.cldr.tool.WikiSubdivisionLanguages exec:java -pl cldr-rdf

**Comment by srl295** [comment]
running the tool yields `cpe` with very many parents, including `tpi`. So i think we're OK!

**Pull request merged**